evalutils/roc.py

```diff
@@ -113,6 +113,66 @@ def get_bootstrapped_roc_ci_curves(
     )
 
 
+def average_roc_curves(
+    roc_curves: List[BootstrappedROCCICurves], bins: int = 200
```

Comment:
```suggestion
    *, roc_curves: List[BootstrappedROCCICurves], bins: int = 200
```

We discussed making keyword arguments required for clarity in use. Do we use this elsewhere in the library as well?

Reply:
As discussed we leave this for now.